backend/app/services/knowledge/knowledge_service.py

```
@@ -1872,9 +1931,150 @@ def get_all_knowledge_bases_grouped(
+        entity_member_group_map: dict[int, str] = {}  # kb_id -> group_name
+        entity_member_role_map: dict[int, str] = {}  # kb_id -> role
+        entity_member_inviter_map: dict[int, int] = {}  # kb_id -> invited_by_user_id
+        entity_member_type_map: dict[int, str] = {}  # kb_id -> entity_type
+        entity_member_entity_id_map: dict[int, str] = {}  # kb_id -> entity_id
+
+        if accessible_groups:
+            # Build namespace name -> id mapping
+            group_namespaces = (
+                db.query(Namespace)
+                .filter(
+                    Namespace.name.in_(accessible_groups), Namespace.is_active == True
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)